### PR TITLE
enable control plane feature to support multi-clusters

### LINKED ISSUE #936: Control plane feature
**Is your feature request related to a problem? Please describe.**
A feature to enable `control-plane` mode.

**Describe the solution you'd like**
When the control plane feature is enabled, the `cluster-worker` cannot be enabled.
The user should be allowed to specify an array of cluster workers to be registered as services.

e.g. 
```
features:
  control-plane:
    enabled: true
    workers:
    - name: minions-1-diego-cell-0
      address: 172.22.216.231
```

See sample here: https://github.com/cloudfoundry-incubator/kubecf/commit/b50a8c8de46e681505621108c440eabd424c469f#diff-50ef3ab48d651f63f084a6815a7115d1

When the `control-plane` feature is enabled, the following ops file must be activated, to remove cells from the control plane:

https://github.com/cloudfoundry-incubator/kubecf/commit/b50a8c8de46e681505621108c440eabd424c469f#diff-8fc2de7ef5a3d25ddfce0cd023890d0cR987-R989

**Describe alternatives you've considered**
n/a

**Additional context**
#822

### PR DESCRIPTION
**Pull request by aqan213**
## Description
To add the control plane feature for multi-clusters when it is enabled, cf components will be deployed except diego-cell.

## Motivation and Context
fixes #936 

## How Has This Been Tested?
Enable the control_plane feature and test in minikube and also IBM IKS clusters.
```
features:
  control_plane:
    enabled: true
```
All cf components are deployed except diego-cell.
```
NAME                                     READY   STATUS      RESTARTS   AGE
adapter-0                                4/4     Running     0          13d
api-0                                    15/15   Running     1          13d
auctioneer-0                             4/4     Running     1          13d
bosh-dns-97764894c-gqfs5                 1/1     Running     0          13d
bosh-dns-97764894c-hqzlz                 1/1     Running     0          13d
cc-worker-0                              4/4     Running     0          13d
database-0                               2/2     Running     1          13d
database-seeder-da4fc217d2879be1-k4nbm   0/2     Completed   0          13d
diego-api-0                              6/6     Running     6          13d
doppler-0                                4/4     Running     0          13d
log-api-0                                7/7     Running     0          13d
log-cache-0                              8/8     Running     0          13d
nats-0                                   4/4     Running     0          13d
router-0                                 5/5     Running     0          13d
routing-api-0                            4/4     Running     0          13d
scheduler-0                              10/10   Running     18         13d
singleton-blobstore-0                    6/6     Running     0          13d
tcp-router-0                             5/5     Running     0          13d
uaa-0                                    6/6     Running     0          13d
```
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: deploy/helm/kubecf/values.yaml [hunk 1]
```diff
@@ -263,6 +263,15 @@ settings:
...
 
 features:
+  # To support multi-clusters, the control_plane is used to deploy cf components except diego-cell. 
+  # While the cluster_worker is to enable diego-cell. When control plane feature is enabled, the cluster-worker cannot be enabled.
+  control_plane:
+    enabled: false
+    workers:
+    - name: minions-1-diego-cell-0
+      address: 172.30.6.59
+  cluster_worker:
```
Comment (viovanov) [inline deploy/helm/kubecf/values.yaml:273]: Shouldn't this match this?

https://github.com/cloudfoundry-incubator/kubecf/pull/1107/files#diff-8fc2de7ef5a3d25ddfce0cd023890d0cR251

Isn't the `cluster_worker` the same as a `cell_segment`?

Comment (aqan213) [reply deploy/helm/kubecf/values.yaml]: @viovanov  I have updated the features to sync with the PR #1107 .Please take a look. Thanks

**Comment by viovanov** [comment]
@aqan213 can you please resolve this conflict?

**Comment by aqan213** [comment]
@viovanov The conflict was resolved.

**Pull request merged**